lib/FusionInventory/Agent/Task/Inventory/Generic/Dmidecode/Memory.pm

```diff
@@ -114,13 +114,8 @@ sub _getMemories {
                     &&
                 $info->{'Part Number'} !~ /
                     DIMM            |
-                    NOT\s*AVAILABLE |
-                    None            |
-                    Not\s*Specified |
-                    O\.E\.M\.       |
```

Comment:
@po1vo I guess here you wanted to filter the string `To Be Filled By O.E.M.`, can you confirm ? This is what I choose in Tools::Generic.

Reply:
Yes, this string and any other alike.

Reply:
I'd leave `O\.E\.M.` to cover more cases.

Reply:
And I'd also remove `^` and `$` from the regexp: in case of leading or trailing characters or whitespaces

Reply:
From getDmidecodeInfos() I can't remove `^` and `$` otherwise many other fields will match on `Unknown` or `<OUT OF SPEC>` like `Present, Unknown` & `Present, <OUT OF SPEC>` for many PSU status.

About `O.E.M.` I only saw 3 cases in our integrated dmidecode outputs: `To Be Filled By O.E.M.`, `    To Be Filled By O.E.M.` (with leading white spaces) & `ToBeFilledByO.E.M.` for a cpu. Then `qr/^\s* To \s* Be \s* Filled \s* By \s* O\.E\.M\.$/xi` as I uses l. 81 just work. Tell us if you have other cases.

Reply:
Seems OK for me. And indeed, it filters out a lot of noise from multiple parts of dmidecode output.